Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Views/Settings/ObjectSettings/Model/ObjectSettingBuilder.swift

```diff
@@ -16,6 +17,10 @@ final class ObjectSettingBuilder {
             }
             
             ObjectSetting.relations
+            
+            if FeatureFlags.versionHistory {
```

Comment:
Participant type has a history. It's possible that if the user is a reader, they also don't have a history. You need to use permissions to display it.

Reply:
Yes, these are just sketches for sure